inc/ruleticket.class.php

```diff
@@ -215,6 +215,21 @@ function executeActions($output, $params, array $input = []) {
                      $output['_'.$action->fields["field"]] = $action->fields["value"];
 
                   }
+
+                  // special case of itil solution template
+                  if ($action->fields["field"] == 'solution_template') {
+                     //load template
+                     $template = new SolutionTemplate();
+                     if ($template->getFromDB($action->fields["value"]) && $output['id'] != null) {
+                        $solution = new ITILSolution();
+                        $solution->add([
+                           "itemtype" => "Ticket",
+                           "content" => $template->getField('content'),
```

Comment:
I guess you should escape content here; add a content with a quote in your tests ;)

Reply:
add code to escape content and TU to test with quote